atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/IndexDefinition.java

```diff
@@ -139,15 +139,15 @@ public void componentFromColumn(String componentName, ValueType valueType, Strin
     }
 
     public void componentFromColumn(String componentName, ValueType valueType, ValueByteOrder valueByteOrder, String sourceColumnName, String codeToAccessValue) {
-        addComponent(IndexComponent.createFromColumn(new NameComponentDescription(componentName, valueType, valueByteOrder), sourceColumnName, codeToAccessValue));
+        addComponent(IndexComponent.createFromColumn(new NameComponentDescription.Builder().componentName(componentName).type(valueType).byteOrder(valueByteOrder).build(), sourceColumnName, codeToAccessValue));
     }
 
     /**
      * Allows multiple index rows when indexing by a cell with iterable values.
      * It doesn't support arbitrary protobuf structures - you need to be able to extract an iterable<valueType> using codeToAccessValue.
      */
     public void componentFromIterableColumn(String componentName, ValueType valueType, ValueByteOrder valueByteOrder, String sourceColumnName, String codeToAccessValue) {
-        addComponent(IndexComponent.createIterableFromColumn(new NameComponentDescription(componentName, valueType, valueByteOrder), sourceColumnName, codeToAccessValue));
+        addComponent(IndexComponent.createIterableFromColumn(new NameComponentDescription.Builder().componentName(componentName).type(valueType).byteOrder(valueByteOrder).build(), sourceColumnName, codeToAccessValue));
```

Comment:
That would have been smart, but actually I had `.Builder().componentName(` in a paste buffer and typed the other method calls by hand!